src/lib.rs

```diff
@@ -14,13 +64,17 @@ pub mod api;
 pub mod index;
 pub mod items;
 
+#[doc(inline)]
 pub use tomldoc::TOMLDocument;
+#[doc(inline)]
 pub use items::*;
 
 // Only public as a hack for testing;
 // Should be private and handled via API
+#[doc(inline)]
 pub mod container;
+#[doc(inline)]
 pub use container::Container;
 
-#[cfg(windows)] pub const NL: &'static str = "\r\n";
-#[cfg(not(windows))] pub const NL: &'static str = "\n";
```

Comment:
Should we remove this? Or did you do this ahead of #32 ?

Reply:
You seem to have a plan for this in your misc branch (?) so I won't worry about it.